source/utils/dynamic-require.ts

```diff
@@ -0,0 +1,3 @@
+export default (moduleId: string, moduleObject: NodeModule = module): unknown => {
```

Comment:
While `= module` works here because it's called in the same package, it's not technically correct, as you're now requiring from the location of this file and not the callers. There should not be a default argument here and `module` should always be required to be passed.

Reply:
I get it. Will fix.